CONTRIBUTING.md

```diff
@@ -13,6 +13,17 @@ V českém jazyce:
 * README a jakákoliv jiná dokumentace
 * Issues a pull requesty
 
+##Schvalování pull requestů
+
+Z celého týmu mají odpovědnost za schvalování PR 
+* Za backend: Michal Valoušek a David Šimek
+* Za frontend: Václav Stejskal a Jiří Petera
+
+Pravidla
+* PR musí schválit vždy jiný člověk než ten co PR vytvořil
+* Informace o novém PR je potřeba napsat do dev kanálu na slacku
+* Veškerá konverzace k PR by měla proběhnout ve vláknu k danému postu ve slacku, případně na github
+
```

Comment:
Zupa, diky 👍 

Dopis tam prosim jeste kdo dela merge do mastera (jestli to je jeden konkretni clovek, nebo vic lidi)

Reply:
@jakubbares Mrkni na to prosim 🔝